build/dependencies.props

```diff
@@ -4,6 +4,7 @@
     <InternalAspNetCoreSdkVersion>2.0.0-*</InternalAspNetCoreSdkVersion>
     <ResponseCompressionVersion>1.1.0-*</ResponseCompressionVersion>
     <AspNetCoreLabsVersion>0.3.0-*</AspNetCoreLabsVersion>
```

Comment:
Correct. I wasn't sure if we were being that aggressive with `dependencies.props`. Happy to remove it.